spec/system/department_admin_should_be_able_to_manage_processes_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Admin manages participatory processes', versioning: true, type: :system do
+  let(:organization) { create(:organization) }
+  let(:area) { create(:area, organization: organization) }
+  let(:department_admin) { create(:department_admin, :confirmed, organization: organization, area: area) }
+
+  let!(:participatory_process_groups) do
+    create_list(:participatory_process_group, 3, organization: organization)
+  end
+
+  before do
+    switch_to_host(organization.host)
+    login_as department_admin, scope: :user
+    visit decidim_admin_participatory_processes.participatory_processes_path
+  end
+
+  # it_behaves_like "manage processes examples"
+  # it_behaves_like "manage processes announcements"
```

Comment:
Remove?

Reply:
These tests are copied from Decidim and then adapted to work with department_admin. I've commented some deeper parts of the spec. For this module tested manually but not with rspec.